back/src/main/java/com/qmate/api/EmailVerificationController.java

```diff
@@ -35,6 +48,10 @@ public ResponseEntity<?> resend(@Valid @RequestBody EmailVerificationSendRequest
   }
 
   @PostMapping("/verify")
+  @Operation(
+      summary = "이메일 인증코드 검증(OK 토큰 발급)",
+      description = EmailVerificationConstants.VERIFY_MD
+  )
   public ResponseEntity<?> verify(@Valid @RequestBody EmailVerificationConfirmRequest req) {
```

Comment:
```suggestion
  public ResponseEntity<EmailConfirmResponse> verify(@Valid @RequestBody EmailVerificationConfirmRequest req) {
```

와일드 카드 사용하면 스웨거문서에 response가 자동으로 매핑되지 않는것 같습니다 이부분 수정이 필요할것 같아요

Reply:
수정했습니다!